mapper/collectd_mapper/src/collectd.rs

```diff
@@ -7,6 +7,44 @@ pub struct CollectdMessage<'a> {
     pub metric_group_key: &'a str,
     pub metric_key: &'a str,
     pub metric_value: f64,
+    pub timestamp: f64,
+}
+
+/// It's easier to deal with owned, sendable data.
```

Comment:
Fully agree on that point. Having a collectd message that references an MQTT, might save some memory but it makes really difficult to parse the message once and for all.

Reply:
Yes, and also this is premature optimization :D